### ISSUE TITLE
Regression since version 4.4.0

### ISSUE THREAD

**Issue by quorak** [fullcalendar/fullcalendar-react#52]
After upgrading to version `4.4.0` i get the following exception, during selecting a timespan in the calender. When reverting back to `4.3.0` it works as usual. 
```
SitesCalendar.js:585 TypeError: Cannot read property 'calendar' of undefined
    at TimeGridSlicer../node_modules/@fullcalendar/core/main.esm.js.Slicer._sliceDateSpan (main.esm.js:8519)
    at TimeGridSlicer.sliceDateSelection (main.esm.js:1552)
    at TimeGridSlicer../node_modules/@fullcalendar/core/main.esm.js.Slicer.sliceProps (main.esm.js:8446)
    at main.esm.js:82
    at mapHash (main.esm.js:1265)
    at ResourceTimeGrid../node_modules/@fullcalendar/resource-timegrid/main.esm.js.ResourceTimeGrid.render (main.esm.js:81)
    at ResourceTimeGrid../node_modules/@fullcalendar/core/main.esm.js.Component.receiveProps (main.esm.js:3951)
    at ResourceTimeGridView../node_modules/@fullcalendar/resource-timegrid/main.esm.js.ResourceTimeGridView.render (main.esm.js:161)
    at ResourceTimeGridView../node_modules/@fullcalendar/core/main.esm.js.Component.receiveProps (main.esm.js:3951)
    at CalendarComponent../node_modules/@fullcalendar/core/main.esm.js.CalendarComponent.renderView (main.esm.js:6298) {componentStack: "↵    in FullCalendar (at SitesCalendar.js:591)↵    in SitesCalendar"}
```

It looks like the https://github.com/fullcalendar/fullcalendar/blame/master/packages/core/src/common/slicing-utils.ts#L168 context refactor commit broke something in combination with how the `fullcaendar-react` component works.

**Comment by ubinatus** [comment]
Hi there! Got the same problem. Had to install 4.3.0 version, as you said. Hoping they would fix this on the v5.

**Comment by Moglum** [comment]
https://codesandbox.io/s/fullcalendar-functional-mydrf
This sandbox demonstrates the error when selecting a timespan. If you change the @fullcalendar dependencies to 4.3.0, the error disappears.

And as a bonus - when you don't use a functional React component, but rewrite it to a Class component, it works even with 4.4.0 🤯
https://codesandbox.io/s/fullcalendar-class-7u48i

**Comment by papiliond** [comment]
+

**Comment by lorinkoz** [comment]
I am wrapping the vanilla javascript version of Fullcalendar inside a functional React component and I'm getting the same problem in v4.4.0, perhaps the problem is not specific to this package.

**Comment by acerix** [comment]
Sorry for the delay in responding to your issue, I have just begun triaging on this repo. Please see this post about the state of the repo (https://github.com/fullcalendar/fullcalendar-react/issues/66) for more info.

I believe this ticket in the main repo is for the same issue which was fixed in version 4.4.1 which was just releasd:
https://github.com/fullcalendar/fullcalendar/issues/5266
